### PR TITLE
added totalParticipantCount to meetingAttendanceReport

### PR DESCRIPTION
**Pull request by frankpeng7**
added totalParticipantCount to meetingAttendanceReport

### CODE REVIEW

**Comment by frankpeng7** [comment]
Hi @isvargasmsft any particular reason this PR was tagged with "do not merge"?

**Comment by isvargasmsft** [comment]
@frankpeng7 I'm about to start the content review for this PR. Please note the **Ready to merge** label should be added once the content review has been completed and the schema published. For more details on the content review process, please visit the summary here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/172563/Content-review-process-summary

File: api-reference/beta/resources/meetingAttendanceReport.md [hunk 1]
```diff
@@ -17,9 +17,10 @@ Contains information associated with meeting attendance report.
 
 ## Properties
 
-| Property            | Type    | Description|
-|:--------------------|:--------|:-----------|
-| attendanceRecords           | [attendanceRecord](attendanceRecord.md) collection  | The list of attendance records. |
+| Property              | Type                                               | Description                     |
+|:----------------------|:---------------------------------------------------|:--------------------------------|
+| totalParticipantCount | Int32                                              | Total number of participants.   |
+| attendanceRecords     | [attendanceRecord](attendanceRecord.md) collection | The list of attendance records. |
```
Comment (isvargasmsft) [inline api-reference/beta/resources/meetingAttendanceReport.md:23]: Please organize the properties alphabetically.

**Review by isvargasmsft — APPROVED** [review]

**Comment by frankpeng7** [comment]
Hi @isvargasmsft the schema has been published. Please help merge this at your earliest convenience. Thanks!

**Review by Lauragra — APPROVED** [review]

**Pull request merged**